scico/numpy/util.py

```diff
@@ -18,6 +19,11 @@
 from ._blockarray import BlockArray
 
 
+def prod(iterable):
```

Comment:
Add a comment here that this can be replaced by `from math import prod` in 3.8+.